plugins/inputs/windows_event_log/wineventlog/wineventlog.go

```diff
@@ -100,6 +114,10 @@ func (w *windowsEventLog) SetOutput(fn func(logs.LogEvent)) {
 	w.startOnce.Do(func() { go w.run() })
 }
 
+func (w *windowsEventLog) SetEventAPI(api WindowsEventAPI) {
```

Comment:
nit: Don't need to expose this function since it's only ever used in this package. Could also just set the `eventAPI` field directly.